highlighters/main/main-highlighter.zsh

```diff
@@ -107,6 +107,9 @@ _zsh_highlight_main_calculate_fallback() {
       dollar-quoted-argument{-unclosed,}
       back-quoted-argument{-unclosed,}
 
+      command-substitution{-quoted,,-unquoted,}
+      command-substitution-delimiter{-quoted,,-unquoted,}
```

Comment:
Thanks for implementing this!  My only comment is that the new styles should be added to the documentation.

Reply:
Indeed. Thanks for the reminder. Fixed.